packages/esm-active-visits-app/package.json

```diff
@@ -15,8 +15,14 @@
     "analyze": "webpack --mode=production --env.analyze=true",
     "lint": "eslint src --ext ts,tsx",
     "typescript": "tsc",
-    "extract-translations": "i18next 'src/**/*.component.tsx'"
+    "extract-translations": "i18next 'src/**/*.component.tsx'",
+    "size": "size-limit"
   },
+  "size-limit": [
+    {
+      "path": "dist/openmrs-esm-active-visits-app.js"
```

Comment:
This is just the entry bundle. I think we want to compare sizes for all the bundles in the module.